metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/MetricsBootListener.java

```diff
@@ -43,13 +42,10 @@ public SlowInvocationLogger getSlowInvocationLogger() {
 
   @Override
   public void onBeforeProducerProvider(BootEvent event) {
-    if (!DynamicPropertyFactory.getInstance().getBooleanProperty("servicecomb.metrics.endpoint.enabled", true).get()) {
+    if (!DynamicPropertyFactory.getInstance().getBooleanProperty("servicecomb.metrics.endpoint.enabled", false).get()) {
```

Comment:
Maybe keep default value to true is better?

Reply:
Under the security sensitive circumstances, the users only want to expose their business REST interfaces. If we keep the default value to be `true`, they may expose these two REST interfaces by mistake.

Reply:
I think these two interfaces do not have any sensitive information. And if the application is secrured , the two interfaces will be secured too. I don't know why explode the two interface will cause security issue.

Reply:
Done, please review